packages/apps/transfer/tsconfig.json

```diff
@@ -6,15 +6,12 @@
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
```

Comment:
Where are we using `js`?

Reply:
Nope, can be removed indeed. (we used to, due to an outdated version of react and some autogenerated files)